include/linux/soundwire/sdw_intel.h

```diff
@@ -450,6 +450,6 @@ extern const struct sdw_intel_hw_ops sdw_intel_lnl_hw_ops;
 /*
  * Max number of links supported in hardware
  */
-#define SDW_INTEL_MAX_LINKS                4
+#define SDW_INTEL_MAX_LINKS                5
```

Comment:
I guess it is OK to accept 5 links on older platforms which only supports 4?
Iow, have ever seen cases that `Link count 5 exceeds max 4` is printed?

Reply:
older platforms are hard-coded to 4, so there's no risk.

But for newer platforms, this could vary over time, so we need to check that the ACPI reported information is aligned with the LCAP register that can be read at probe time.

will add a new patch.